css/ExpandableListItem.less

```diff
@@ -27,6 +27,9 @@
 .moon-expandable-list-item-client.indented {
 	padding-left: @moon-item-indent + @moon-spotlight-outset;
 }
+.enyo-locale-right-to-left .moon-expandable-list-item-header {
```

Comment:
Hi @anishlg, 
We usually doesn't encourage to use '!important' in general. Instead, I think we can use a more specific css rule like '.enyo-locale-right-to-left .moon-expandable-list-header.moon-expandable-list-item-header'.
How do you think?